layouts/jobs/list.html

```diff
@@ -1,15 +1,53 @@
 {{ define "main" }}
-<section class="grid grid-cols-3 grid-rows-4">
+<section class="grid grid-cols-3 grid-rows-4 gap-3">
 	<div class="col-span-2">
 		{{ .Content }}
 	</div>
 	<div class="row-span-2">
 		<h1>Character image</h1>
 		<img src="" />
 	</div>
-	<div class="row-span-2"><h1>Job Fundamentals</h1></div>
-	<div class="row-span-2"><h1>Basics</h1></div>
+	<div class="row-span-2">
+		<h1>Job Fundamentals</h1>
+		<div>
+			{{ with .Site.GetPage "leveling-guide" }}
```

Comment:
I could be misunderstanding the Hugo documentation ([link for reference](https://gohugo.io/functions/getpage/)) but if the intention is to use a relative path I think we need to use the .Page not the .Site variant of the function.